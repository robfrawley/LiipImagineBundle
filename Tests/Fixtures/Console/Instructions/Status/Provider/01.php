<?php

/*
 * This file is part of the `liip/LiipImagineBundle` project.
 *
 * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
 *
 * For the full copyright and license information, please view the LICENSE.md
 * file that was distributed with this source code.
 */

use Liip\ImagineBundle\Console\Style\Style;
use Liip\ImagineBundle\Tests\Fixtures\Console\FixtureProvider;

return function (Style $style, FixtureProvider $fixture) {
    foreach ($fixture->data('status', 'results') as $s) {
        $style->status($s)->newline(2);
    }
};
