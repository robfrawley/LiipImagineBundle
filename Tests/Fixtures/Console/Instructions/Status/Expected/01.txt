(<options=bold>resolved</>)

(<options=bold>skipped</>)

(<options=bold>failed</>)

(<options=bold>cached</>)

(<options=bold>ignored</>)

